data/categories.txt

```diff
@@ -6093,67 +6093,688 @@ sw:Bwawa la kuogelea
 fa:استخرشنا
 mr:जलतरण तलाव
 
-sport-equestrian
-en:Equestrian Sports
-ru:Конный спорт
-bg:Коне|спорт|езда
-nl:Paardensport|ruitersport
-ko:승마|마술 스포츠
-pl:Jazda konna
-fr:Sport équestre
-hu:Lovassportok
-de:Pferdesport
-it:Equitazione
-es:Deportes ecuestres
-eu:Zaldi-kirolak
-pt:Desportos equestres
-pt-BR:Esportes equestres
-tr:Binicilik
-el:Ιππασία
-mr:घोडेस्वार खेळ
+sport-american_football
+en:American Football
+ar:كرة القدم الأمريكية
+be:Амерыканскі футбол
+cs:Americký fotbal
+da:Amerikansk fodbold
+de:Amerikanischer Fußball
+el:Αμερικάνικο ποδόσφαιρο
+es:Fútbol americano
+es-MX:Fútbol americano
+eu:Futbol amerikarra
+fa:فوتبال آمریکایی
+fi:Amerikkalainen jalkapallo
+fr:Football américain
+he:כדורגל אמריקאי
+hu:Amerikai foci
+id:Sepak Bola Amerika
+it:Football americano
+ja:アメリカンフットボール
+ko:미식 축구
+mr:अमेरिकन फुटबॉल
+nb:Amerikansk fotball
+nl:Amerikaans voetbal
+pl:Futbol amerykański
+pt:Futebol americano
+pt-BR:Futebol americano
+ro:Fotbal american
+ru:Aмериканский футбол
+sk:Americký futbal
+sv:Amerikansk fotboll
+sw:Soka ya Marekani
+th:อเมริกันฟุตบอล
+tr:Amerikan futbolu
+uk:Американський футбол
+vi:Bóng bầu dục Mỹ
+zh-Hans:美式足球
+zh-Hant:美式足球
+
+sport-archery
+en:Archery
+ar:الرماية
+be:Стральба з лука
+cs:Lukostřelba
+da:Bueskydning
+de:Bogenschießen
+el:Τοξοβολία
+es:Tiro al arco
+es-MX:Tiro al arco
+eu:Arku-tiroa
+fa:تیراندازی با کمان
+fi:Jousiammunta
+fr:Tir à l'arc
+he:קַשׁתוּת
+hu:Íjászat
+id:Panahan
+it:Tiro con l'arco
+ja:アーチェリー
+ko:양궁
+mr:धनुर्विद्या
+nb:Bueskyting
+nl:Boogschieten
+pl:Łucznictwo
+pt:Tiro com arco
+pt-BR:Tiro com arco
+ro:TIR cu arcul
+ru:Стрельба из лука
+sk:Lukostreľba
+sv:Bågskytte
+sw:Upigaji mishale
+th:ยิงธนู
+tr:Okçuluk
+uk:Стрільба з лука
+vi:Bắn cung
+zh-Hans:射箭
+zh-Hant:射箭
+
+sport-athletics
+en:Athletics
+ar:ألعاب القوى
+be:Лёгкая атлетыка
+bg:атклетика|лека
+cs:Atletika
+da:Atletik
+de:Leichtathletik
+el:Στίβος
+es:Atletismo
+eu:Atletismoa
+fa:ورزشکاری
+fi:Yleisurheilu
+fr:Athlétisme
+he:אַתלֵטִיקָה
+hu:Atlétika
+id:Atletik
+it:Atletica leggera
+ja:陸上競技
+ko:체육시설
+mr:शरीरिक कसरती|ऍथलेटिक्स
+nb:Friidrett
+nl:Atletiek
+pl:Lekkoatletyka
+pt-BR:Atletismo
+pt:Atletismo
+ro:Atletism
+ru:Лёгкая атлетика
+tr:Atletik
+uk:Легка атлетика
+vi:Thế vận hội
+zh-Hans:竞技
+zh-Hant:競技
+
+sport-austrialian_football
+en: Rugby
```

Comment:
I checked a wikipedia article on Australian Football and its not Rugby, its a separate kind of sport!